### ISSUE TITLE
Public link not public

### ISSUE THREAD

**Issue by rmc0** [zusam/zusam_ynh#3]
I think Public link is block by yunohost so only yunohost user can access to public link after login yunohost

**Comment by nrobinaubertin** [comment]
It's actually a zusam bug. I've fixed it (https://github.com/zusam/zusam/commit/518a3b82075444e1d87b8e7da67727e950121585), it will be shipped with 0.4.3

**Comment by rmc0** [comment]
0.4.3 update crash zusam, only white page 
after remove and install zusam,

 public link doesn't open tab like before, I can see 1second the url with public  and the page auto reload to groups view

**Comment by nrobinaubertin** [comment]
Yes I'm aware that it doesn't work on Yunohost. I started to check it after the release.  
I'm sorry, life is a bit getting in the way at the moment. I'll fix it asap.

I should add to the readme that yunohost support is alpha/experimental for now

**Comment by nrobinaubertin** [comment]
I've fixed the install/upgrade scripts of the ynh zusam package but since it uses the upgrade script of the current version that you have, you will need to reinstall to have the correct one.  
I'm taking steps to be more serious about testing this package in the future and be more transparent about it's status. Sorry for the issues.

I identified and fixed the bug for the public link on Zusam's side and will ship the definitive fix with the next version (this month probably).  
Thanks for your feedback.
